io.qt.qt5-base.json

```diff
@@ -0,0 +1,76 @@
+{
```

Comment:
This was mentioned to you already but you should just use the KDE runtime. Their build of Qt has a dozen Flatpak specific patches and is actually well maintained by Qt contributors. This is not something we want packages to reproduce.

Reply:
It's using the KDE runtime now, as I really don't want to maintain Qt on my own indeed.